### PR TITLE
feat(components): InputAvatar & Expand variation options for InputFile

### PR DESCRIPTION
**Pull request by seanhealy**
### Added

- Adds InputAvatar Component
- Adds Button only mode for InputFile
- Adds Size and Variation options to InputFile

### Changed

- 

### Deprecated

- 

### Removed

- 

### Fixed

- 

### Security

- 

## Testing

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

Random photo of Atlantis

### CODE REVIEW

File: packages/components/src/InputFile/InputFile.css [hunk 1]
```diff
@@ -14,3 +19,7 @@
   border-color: var(--border-active-color);
   background-color: var(--color-yellow--lightest);
 }
+
+.input {
+  margin: 0;
```
Comment (eddysims) [inline packages/components/src/InputFile/InputFile.css:24]: Is the `input` in `InputFile` not `display: none`? do we need this line here?

**Review by eddysims — CHANGES_REQUESTED** [review]

File: packages/components/src/FormatFile/FormatFile.css [hunk 2]
```diff
@@ -34,6 +34,19 @@
   object-fit: cover;
   flex-shrink: 0;
 }
+.imageBlock::before {
```
[the remark this answers is not in the document]
Comment (seanhealy) [reply packages/components/src/FormatFile/FormatFile.css:37]: YES

File: packages/components/src/InputAvatar/InputAvatar.css [hunk 3]
```diff
@@ -0,0 +1,42 @@
+.inputAvatar {
+  display: flex;
+  align-items: center;
+}
+
+.inputAvatar > *:not(:last-child) {
```
[the remark this answers is not in the document]
Comment (seanhealy) [reply packages/components/src/InputAvatar/InputAvatar.css:6]: It needs to apply to each element in that row depending on how many there are was my thought.

File: packages/components/src/InputAvatar/InputAvatar.test.tsx [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+    expect(changeHandler).toHaveBeenCalledWith({
+      key: "atlantis.png",
+      name: "atlantis.png",
+      size: expect.any(Number),
+      progress: expect.any(Number),
+      src: expect.any(Function),
+      type: "image/png",
+    });
+  });
+});
```
[the remark this answers is not in the document]
Comment (eddysims) [reply packages/components/src/InputAvatar/InputAvatar.test.tsx:60]: 🥇 

image

File: packages/components/src/InputFile/InputFile.tsx [hunk 5]
```diff
@@ -101,6 +119,9 @@ interface InputFileProps {
 }
 
 export function InputFile({
+  variation = "dropzone",
+  size = "base",
+  buttonLabel: providedButtonLabel,
```
Comment (eddysims) [inline packages/components/src/InputFile/InputFile.tsx:124]: A lack of knowledge here, how does this work? what is `providedButtonLabel`?

File: packages/components/src/InputFile/InputFile.tsx [hunk 6]
```diff
@@ -115,23 +136,40 @@ export function InputFile({
...
   const { getRootProps, getInputProps, isDragActive } = useDropzone(options);
 
-  const { buttonLabel, hintText } = getLabels(allowMultiple, allowedTypes);
-  const dropZone = classnames(styles.dropZone, {
+  const { buttonLabel, hintText } = getLabels(
+    providedButtonLabel,
+    allowMultiple,
+    allowedTypes,
+  );
+  const dropZone = classnames(styles.dropZoneBase, {
```
Comment (eddysims) [inline packages/components/src/InputFile/InputFile.tsx:144]: I wish we could add something to eslint to force a blank line after declaration.

**Review by eddysims — APPROVED** [review]
@seanhealy looks good to me, I have a couple questions but not blockers.

File: packages/components/src/InputFile/InputFile.tsx (hunk 5 above)
Comment (seanhealy) [reply packages/components/src/InputFile/InputFile.tsx:124]: That renames the value that it is destructured into.

**Pull request merged**